regression/rt_gk_neut_sheath_3x2v_p1.c

```diff
@@ -0,0 +1,1218 @@
+    .num_integrated_diag_moments = 1,
+    .integrated_diag_moments = { "HamiltonianMoments" },
+    .time_rate_diagnostics = true,
+
+    .boundary_flux_diagnostics = {
+      .num_integrated_diag_moments = 1,
+      .integrated_diag_moments = { "HamiltonianMoments" },
+    },
+  };
+
+
+  struct gkyl_gyrokinetic_neut_species D0 = {
+    .name = "D0",
+    .mass = ctx.mass_ion,
+    .lower = { -ctx.vmax_neut, -ctx.vmax_neut, -ctx.vmax_neut},
+    .upper = { ctx.vmax_neut, ctx.vmax_neut, ctx.vmax_neut },
+    .cells = { NMU, NMU, NMU },
+    .is_static = true,
+    
+    .projection = {
+      .proj_id = GKYL_PROJ_MAXWELLIAN_PRIM, 
+      .ctx_density = &ctx,
+      .density = neut_density_init,
+      .ctx_upar = &ctx,
+      .udrift= udrift,
+      .ctx_temp = &ctx,
+      .temp = temp_neut,      
+    },
+
+    /* .init_from_file = { */
+    /*   .type = GKYL_IC_IMPORT_F, */
+    /*   .file_name = "D0_restart.gkyl", */
+    /* }, */
+
+    .react_neut = {
+      .num_react = 3,
+      .react_type = {
+        { .react_id = GKYL_REACT_IZ,
+          .type_self = GKYL_SELF_DONOR,
+          .ion_id = GKYL_ION_H,
+    	  .elc_nm = "elc",
+          .ion_nm = "ion",
+          .donor_nm = "D0",
+    	  .charge_state = 0,
+          .ion_mass = ctx.mass_ion,
+          .elc_mass = ctx.mass_elc,
+        },
+    	{ .react_id = GKYL_REACT_RECOMB,
+          .type_self = GKYL_SELF_RECVR,
+          .ion_id = GKYL_ION_H,
+    	  .elc_nm = "elc",
+          .ion_nm = "ion",
+          .donor_nm = "D0",
+    	  .charge_state = 0,
+          .ion_mass = ctx.mass_ion,
+          .elc_mass = ctx.mass_elc,
+        },
+    	{ .react_id = GKYL_REACT_CX,
+          .type_self = GKYL_SELF_PARTNER,
+          .ion_id = GKYL_ION_H,
+          .ion_nm = "ion",
+          .partner_nm = "D0",
+          .ion_mass = ctx.mass_ion,
+          .partner_mass = ctx.mass_ion,
+        },
+      },
+    },
+
+    .bcx = {
+      .lower = {.type = GKYL_SPECIES_ABSORB,},
+      .upper = {.type = GKYL_SPECIES_ABSORB,},
+    },
+    .bcz = {
+      .lower = {
+        .type = GKYL_SPECIES_ABSORB,
+      },
+      .upper = {
+        .type = GKYL_SPECIES_ABSORB,
+      },
+    },
+//    .bcz = {
+//      .lower = {
+//        .type = GKYL_SPECIES_RECYCLE,
+//        .emission = neut_bc,
+//        .projection = {
+//          .proj_id = GKYL_PROJ_MAXWELLIAN_PRIM,
+//          .ctx_density = &ctx,
+//          .density = unit_density,
+//          .ctx_upar = &ctx,
+//          .udrift= udrift,
+//          .ctx_temp = &ctx,
+//          .temp = temp_neut,
+//        },
+//      },
+//      .upper = {
+//        .type = GKYL_SPECIES_RECYCLE,
+//        .emission = neut_bc,
+//        .projection = {
+//          .proj_id = GKYL_PROJ_MAXWELLIAN_PRIM,
+//          .ctx_density = &ctx,
+//          .density = unit_density,
+//          .ctx_upar = &ctx,
+//          .udrift= udrift,
+//          .ctx_temp = &ctx,
+//          .temp = temp_neut,
+//        },
+//      },
+//    },
```

Comment:
@tnbernard do you still want these to be commented out? if so, can we just delete them?

Reply:
I have updated this file to include recycle bcs.